### PR TITLE
community/colord-gtk: upgrade to 0.2.0

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
@Ikke

File: community/colord-gtk/APKBUILD [hunk 1]
```diff
@@ -1,42 +1,35 @@
 # Contributor: Kevin Daudt <ops@ikke.info>
 # Maintainer: Kevin Daudt <ops@ikke.info>
 pkgname=colord-gtk
-pkgver=0.1.26
-pkgrel=2
+pkgver=0.2.0
+pkgrel=0
 pkgdesc="GTK support library for colord"
+options="!check" # Requires display
```
Comment (Cogitri) [inline community/colord-gtk/APKBUILD]: xvfb-run doesn't work here?

Comment (maxice8) [reply community/colord-gtk/APKBUILD]: no

**Comment by Ikke** [comment]
> @Ikke

As long as darktable works with this version, it's fine with me.

**Pull request closed without merging**